src/vs/workbench/workbench.common.main.ts

```diff
@@ -374,6 +374,7 @@ import 'vs/workbench/contrib/surveys/browser/languageSurveys.contribution';
 // Welcome
 import 'vs/workbench/contrib/welcome/overlay/browser/welcomeOverlay';
 import 'vs/workbench/contrib/welcome/page/browser/welcomePage.contribution';
+import 'sql/workbench/contrib/welcome/page/browser/welcomePage.contribution'; // {{SQL CARBON EDIT}} - SQL welcome page contribution
```

Comment:
we should keep all the sql imports together in this file.  I think there is a section for SQL imports below.  Please make this change before merging this PR.